src/jqLite.js

```diff
@@ -284,7 +284,7 @@ function JQLite(element) {
   }
   if (!(this instanceof JQLite)) {
     if (argIsString && element.charAt(0) !== '<') {
-      throw jqLiteMinErr('nosel', 'Looking up elements via selectors is not supported by jqLite! See: http://docs.angularjs.org/api/angular.element');
+      return new JQLite(document.querySelector(element));
```

Comment:
You need to use `window.document`. We don't assume browser globals are available globally, they have to be taken from `window`. You can also see that the Travis build failed because of that.

Reply:
I added the `window` call to it and rebased everything.